js/main.js

```diff
@@ -8,6 +10,28 @@ search.addEventListener('click', function (event) {
   xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + input);
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
+    if (xhr.status === 200) {
+      $searchAddMember.classList.remove('hidden');
```

Comment:
Somewhere in the function you should set your form input value to an empty `''` to clear it out instead of refreshing the browser to clear it out.